chat-plugins/mafia.js

```diff
@@ -1102,35 +1368,47 @@ exports.commands = {
 
 		'!start': true,
 		start: function (target, room, user) {
-			let targetRoom = room;
-			target = target.split(',');
-			if (Rooms(target[0]) && Rooms(target[0]).users[user.userid]) targetRoom = Rooms(target.shift());
-			if (!targetRoom || !targetRoom.game || targetRoom.game.gameid !== 'mafia') return this.errorReply(`There is no game of mafia running in this room.`);
-			if (!user.can('mute', null, room) && targetRoom.game.hostid !== user.userid) return user.sendTo(targetRoom, `|error|/mafia start - Access denied.`);
-			targetRoom.game.start(user);
+			const args = target.split(',');
+			let targetRoom /** @type {ChatRoom?} */ = (Rooms(args[0]));
+			if (!targetRoom || targetRoom.type !== 'chat' || !targetRoom.users[user.userid]) {
+				if (!room || room.type !== 'chat') return this.errorReply(`This command is only meant to be used in chat rooms.`);
+				targetRoom = room;
+			} else {
+				args.shift();
```

Comment:
dont need it for this

Reply:
Lack of perception while refactoring x.x